src/scss/components/optional/_ratings.scss

```diff
@@ -18,9 +21,12 @@
     $star-size--medium          : 28px;
     $star-size--large           : 42px;
 
+  // https://github.com/justeat/f-icons/blob/master/icons/star-filled.svg
+  @warn 'To use the rating component you will need to add references to the SVGs in f-icons';
 
     .c-rating {
-        @extend %c-icon--star--empty !optional;
+        // Leaving this @extend in so people can port over from the old icon repo easily.
```

Comment:
I've just come across this (missed it the first time), what is the guidance on getting the star ratings into projects now?

Reply:
@DamianMullins myself and @ashleynolan will catch up with you about this. I will also document it once we've decided on how to resolve it. 👍